frontend/src/components/form/api/ApiWrapper.vue

```diff
@@ -82,8 +80,17 @@ export default {
       if (this.hasServerError) errors.push(this.serverErrorMessage)
       return errors
     },
+    isSaving() {
+      return this.savingRequestCount > 0
+    },
+    hasLoadingError() {
+      return this.loadingErrorMessage != null || this.loadingErrorMessage === ''
```

Comment:
`this.loadingErrorMessage` is initialized with `''`. 
`hasLoadingError()` is therefore initial ` == true` 
seems a bit strange to me. Is that intentional?